components/core/TestCommands.py

```diff
@@ -0,0 +1,46 @@
+import unittest
```

Comment:
What does this file do? is it a script?

Reply:
this file adds a command for running the tests without the coverage and with the coverge the respective commands being `python TestCommands.py test` and `python TestCommands.py cov`